spec/rubocop/cop/packaging/relative_require_to_lib_spec.rb

```diff
@@ -0,0 +1,106 @@
+# frozen_string_literal: true
+
+RSpec.describe RuboCop::Cop::Packaging::RelativeRequireToLib do
+  subject(:cop) { described_class.new(config) }
+
+  let(:config) { RuboCop::Config.new }
+
+  let(:message) { RuboCop::Cop::Packaging::RelativeRequireToLib::MSG }
+
+  let(:root) { RuboCop::ConfigLoader.project_root }
+
+  let(:processed_source) { parse_source(source) }
+
+  before do
+    allow(processed_source.buffer)
+      .to receive(:name).and_return(filename)
+    _investigate(cop, processed_source)
+  end
+
+  context 'when `require_relative` call lies outside spec/' do
+    let(:filename) { "#{root}/spec/foo_spec.rb" }
+    let(:source) { 'require_relative "../lib/foo.rb"' }
```

Comment:
All the specs fail, even though the implementation is working fine, as it should.
(I locally installed this version and tested that)

No clue about why the specs fail. @deivid-rodriguez, any idea?

Reply:
Hi @utkarsh2102! The problem seems indeed with the specs, since they are using `Rubocop::ConfigLoader.project_root` just like the implemetation. However, in the specs things are mocked, there's no such `spec/foo_spec.rb` file and so on. So, go back to the previous implementation of the specs where we were mocking the project root to be `/some`. You could do something like `allow(Rubocop::ConfigLoader.to receive(:project_root).and_return("/some")`.